docs/userGuide/tweakingThePageStructure.md

```diff
@@ -148,20 +147,46 @@ If you wish insert scripts at the bottom, before MarkBind's scripts, simply inse
 
 ---
 
-### Fixing the header to the top
+### Sticking the header to the top
 
-Headers are commonly included inside the HTML `<header>` tag. In encouraging this, a convenient interface to implement <tooltip content="Headers that stick to the top of the page while scrolling the content">fixed headers</tooltip> surrounding the `<header>` tag is provided that ensures page anchors work correctly.
+Headers are commonly included inside the html `<header>` tag. In encouraging this, 2 convenient options to implement fixed/sticky headers using the `<header>` tag are provided, which primarily ensures that the page **scrolls to anchors** correctly.
 
-****To fix the `<header>`****
+#### Option 1: Fixed header
+
+****Steps****
 1. Add the `fixed` attribute to your `<header>` element in the layout per the above example.
 
-2. Then, to add the necessary top padding for the main content, add the `fixed-header-padding` class to **elements that should be shifted down** in accordance with the fixed header.
+2. Then, to add the necessary top padding for the main content, add the `fixed-header-padding` class to **elements that should be shifted down** in accordance with the height of the fixed header.
+   
+   This class should generally be added to the elements that immediately below the header. You may also refer to the default template generated by `markbind init`, or the above code example (which is used for this documentation).
+
+<box type="warning" seamless>
+
+As this feature does not make any assumptions of your header height, runtime height detection is performed to populate the `fixed-header-padding` class. This results in improperly formatted content on page load for a brief period.
+
+To avoid this, the implementation here switches the `<header>` element to use `fixed` positioning, and populates the `fixed-header-padding` class's padding *only once* the user has scrolled past the top of the header. Thus, ensure your `<header>` element is the first visible element.
+</box>
```

Comment:
May I check what you intend to explain here?

> As this feature does not make any assumptions of your header height, runtime height detection is performed to populate the `fixed-header-padding` class. This results in improperly formatted content on page load for a brief period.

This says that using the `fixed-header-padding` class will result in improperly formatted content... but
> To avoid this, the implementation here switches the `<header>` element to use `fixed` positioning, and populates the `fixed-header-padding` class's padding *only once* the user has scrolled past the top of the header. Thus, ensure your `<header>` element is the first visible element.

does this say that it actually won't? So is there anything that the user needs to do if he/she chooses to go with option 1 with `fixed` + `fixed-header-padding`?

Perhaps can rephrase this section to make it clearer:)